src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/missingswitchdefault/InputMissingSwitchDefault.java

```diff
@@ -19,4 +19,29 @@ public void foo() {
         case 2: i--; break;
```

Comment:
this switch also produces violation, please add comment here too as well.

Reply:
OK